examples/with-realm-web/lib/RealmClient.js

```diff
@@ -0,0 +1,67 @@
+//Realm
+import { Credentials, App } from 'realm-web'
+//Apollo
+import { ApolloClient } from 'apollo-client'
```

Comment:
Can we switch to do a simple fetch (using SWR would be great) instead of Apollo? I would like to focus more on `realm` and ignore heavy packages that can be replaced with a simpler alternative.

Reply:
done in 125f149